### ISSUE TITLE
CODE Server NextCloud - formula separator bug on other locale

### ISSUE THREAD

**Issue by Screamer89** [CollaboraOnline/online#8955]
Because the current formula separator settings conflict with the locale, the formula separators have been reset to their default values

**Comment by Kratzi** [comment]
I have the same error each time I created a new spreadsheet. I got an additional error after downloading the created spreadsheet and open it with LibreOffice 7.3.

The error is gone after changing my user language to English.

User language: German
Nextcloud: 28.0.5
Collabora Online Development Edition: 24.04.1.3 

I'm using the integrated CODE Edition for my Nextcloud.

Spreadsheet error:
![Screenshot from 2024-05-03 10-13-33](https://github.com/CollaboraOnline/online/assets/26336836/2af9e310-a284-44a5-af53-4e550ae58671)

LibreOffice error: 
![Screenshot from 2024-05-03 10-18-34](https://github.com/CollaboraOnline/online/assets/26336836/3353cd8b-cc55-4a48-8984-84e0e2a0c421)
